packages/marketplace/src/components/ui/developer-app-detail/app-header.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react'
+import { H2 } from '@reapit/elements'
+import { FaCheck } from 'react-icons/fa'
+import { AppDetailModel } from '@reapit/foundations-ts-definitions'
+import styles from '@/styles/blocks/standalone-app-detail.scss?mod'
+
+export type AppHeaderProps = {
+  appDetailData: AppDetailModel & {
+    apiKey?: string | undefined
+  }
+  buttonGroup?: React.ReactNode
+}
+
+const appIconClassName = ['images', styles.appIcon].join(' ')
+
+const placeHolderUrl = 'https://bulma.io/images/placeholders/48x48.png'
```

Comment:
should we bring the image to our repo?

Reply:
@tanphamhaiduong  I removed it already, we no longer need to use that placeholder image